src/hip_hcc.cpp

```diff
@@ -1561,7 +1567,7 @@ void ihipPrintKernelLaunch(const char* kernelName, const grid_launch_parm* lp,
 // Allows runtime to track some information about the stream.
 hipStream_t ihipPreLaunchKernel(hipStream_t stream, dim3 grid, dim3 block, grid_launch_parm* lp,
                                 const char* kernelNameStr) {
-    HIP_INIT();
+    hip_impl::hip_init();
```

Comment:
Internal functions shouldn't call hip_init(), caller of this function should have already initialized the HIP.

Reply:
removed